core/include/userver/server/http/http_response_cookie.hpp

```diff
@@ -60,9 +61,10 @@ class Cookie final {
   std::string SameSite() const;
   Cookie& SetSameSite(std::string value);
 
-  std::string ToString() const;
+  USERVER_NAMESPACE::http::headers::HeadersString ToString() const;
```

Comment:
I'm uncertain about this: now instead of 24/32 bytes object we return a whopping 1024 bytes object.

Moving/copying that could be extremely inefficient somewhere in the caller